scripts/west_commands/runners/nrfjprog.py

```diff
@@ -33,19 +34,22 @@ def has_region(regions, hex_file):
     except FileNotFoundError:
         return False
 
+# https://infocenter.nordicsemi.com/index.jsp?topic=%2Fug_nrf_cltools%2FUG%2Fcltools%2Fnrf_nrfjprogexe_return_codes.html&cp=9_1_3_1
+UnavailableOperationBecauseProtectionError = 16
```

Comment:
@simtind this is what we should check for, right?

Reply:
Yes